ticktick_mcp/src/server.py

```diff
@@ -429,6 +429,49 @@ async def delete_project(project_id: str) -> str:
         logger.error(f"Error in delete_project: {e}")
         return f"Error deleting project: {str(e)}"
 
+@mcp.tool()
+async def create_subtask(
+    subtask_title: str,
+    parent_task_id: str,
+    project_id: str,
+    content: str = None,
+    priority: int = 0
```

Comment:
As I reviewed changes and TickTick API documentation, I acknowledged that priority should be integer values : 0, 1, 3, 5. I have thought it might be a good improvement to expose the priority argument as an enum or a Literal type.

Such as,
```python
from typing import Literal

priority: Literal["Low", "Medium", "High"] | None
```